sphinx_gallery/tests/test_full.py

```diff
@@ -849,21 +855,26 @@ def _rerun(
     # ... but then later detects that only some have actually changed:
     lines = [line for line in status.split("\n") if "changed," in line]
     # Ones that can change on stale:
+    #
     # - auto_examples/future/plot_future_imports_broken
     # - auto_examples/future/sg_execution_times
     # - auto_examples/plot_scraper_broken
     # - auto_examples/sg_execution_times
+    # - auto_examples_rst_index/sg_execution_times
+    # - auto_examples_with_rst/sg_execution_times
     # - sg_api_usage
-    # Sometimes it's not all 5, for example when the execution time and
+    # - sg_execution_times
+    #
+    # Sometimes it's not all 8, for example when the execution time and
     # memory usage reported ends up being the same.
     #
     # Modifying an example then adds these two:
     # - auto_examples/index
     # - auto_examples/plot_numpy_matplotlib
     if how == "modify":
-        n_ch = "[3-7]"
+        n_ch = "([3-9]|10)"
```

Comment:
Why not `[3-10]` ?

Reply:
I had that originally! But it's bad regex :) `[3-9]` is a character range. If you write `[3-10]` regex complains because it sees this as "the range of characters from 3-1, and also the character zero" -- and the range `3-1` doesn't make sense (it would need to be 1-3 and it's also not what we want). So by doing `([3-9]|10)` it will match any single digit between 3 and 9, *or* the literal string 10, which is what we want.

Reply:
Ah yes of course, thanks!